etc/kayobe/kolla.yml

```diff
@@ -102,6 +147,12 @@ kolla_openstack_release: "6.0.0.0"
 # most commonly packages. The operation should be one of override, append or
 # remove. The value should be a list.
 #kolla_build_customizations:
+kolla_build_customizations:
+   # Use our own repo server to install the monasca-grafana fork
+   base_yum_repo_files_remove:
+     - 'grafana.repo'
+   influxdb_packages_override:
+     - 'influxdb-0.10.3-1'
```

Comment:
Will we also require a kafka version override, when that comes through?

Reply:
Pass. I took these from alaska & VG.